lib/code_corps_web/controllers/password_controller.ex

```diff
@@ -9,6 +9,7 @@ defmodule CodeCorpsWeb.PasswordController do
   """
   def forgot_password(conn, %{"email" => email}) do
     ForgotPasswordService.forgot_password(email)
+    conn = Guardian.Plug.sign_out(conn)
```

Comment:
@begedin having trouble testing this.  Looking in the `conn`, I see differences in the `password_controller_test` when this line is here, but haven't figured out how to verify.

Reply:
Maybe check to see how `Guardian` tests? https://github.com/ueberauth/guardian/blob/d65227145f72b290106c06cecbe42728fbf05fe2/test/guardian/plug_test.exs#L255

Reply:
So looking at this a second time, I am not seeing a sol'n yet.

`guardian_default_jwt` is still on the `conn` after posting to the forgot password controller. Hmmm.

Reply:
Is `Guardian.Plug.authenticated?(conn)` behaving correctly? As in, returns `true` on the "before" conn, `false` on the "after"? 

If that is the case, instead of worrying about the internals, I'd just use that.

Reply:
password_controller.ex
```
    IO.inspect Guardian.Plug.authenticated?(conn)
    conn = Guardian.Plug.sign_out(conn)
    IO.inspect Guardian.Plug.authenticated?(conn)
```

Both return `true`.  So I'm thinking `sign_out` isn't doing what I think.

Reply:
Looks like Guardian is changing under the hood as well with some breaking changes: https://github.com/ueberauth/guardian/blob/f397888943629c487d0b6f9ee1ba6ec5d104fa8b/upgrade_guides/0.14.to.1.0.md

Reply:
Ah figured it out.  Did some debugging and got down to the issue.  @begedin @JoshSmith pending your review, this PR should be g2g!